src/studiolibrary/librarywidget.py

```diff
@@ -103,6 +104,13 @@ class LibraryWidget(QtWidgets.QWidget):
     folderRenamed = QtCore.Signal(str, str)
     folderSelectionChanged = QtCore.Signal(object)
 
+    # Customize widget classes
+    ITEMSWIDGETCLASSTYPE = studioqt.ItemsWidget
```

Comment:
Thanks for adding these. Can we use underscores to separate the words and remove TYPE?
eg:

ITEMS_WIDGET_CLASS = studioqt.ItemsWidget

Reply:
Please don't worry about the naming in this pull request.